client/blocks/get-apps/apps-card.tsx

```diff
@@ -0,0 +1,30 @@
+import { Card, CardBody } from '@wordpress/components';
+import { PropsWithChildren, FC } from 'react';
+import SVGIcon from 'calypso/components/svg-icon';
+
+type AppsCardProps = PropsWithChildren< {
+	logo: string;
+	logoName: string;
+	title: string;
+	subtitle: string;
+} >;
+
+export const AppsCard: FC< AppsCardProps > = ( { logo, logoName, title, subtitle, children } ) => {
+	return (
+		<Card isRounded={ false } className="get-apps__card">
+			<CardBody style={ { padding: '24px' } }>
+				<SVGIcon
+					name={ logoName }
+					aria-hidden="true"
+					size={ 64 }
+					viewBox="0 0 64 64"
+					icon={ logo }
+					classes="get-apps__card-logo"
+				/>
+				<h2 className="get-apps__card-title">{ title }</h2>
+				<p className="get-apps__card-subtitle">{ subtitle }</p>
+				{ children && <div className="get-apps__card-content">{ children }</div> }
```

Comment:
It would be ideal if we could find a way to align `.get-apps__card-content` to the bottom of the container. It would just look nicer if the download buttons aligned properly. Not trivial because we are using WP components, but worth looking into IMO. cc @matt-west 

![calypso localhost_3000_me(Medium laptop)](https://github.com/user-attachments/assets/11e7098e-a03f-4263-a859-58f35c190ced)